components/ActionModals/TableInvoice.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <FormGroup
-      v-if="getDenoms.length > 1"
+      v-if="getDenoms.length > 1 && amounts.length > 1"
```

Comment:
nope. you should be able to pay with any of the available fees

Reply:
If there is only one amount (one denom being transacted), it makes sense to default to that denom to pay the fee. Ah yeah, but you could still be able to choose.

OK, changing this